empyrical/stats.py

```diff
@@ -739,11 +976,11 @@ def stability_of_timeseries(returns):
     if len(returns) < 2:
         return np.nan
 
-    returns = returns.dropna()
+    returns = returns[~np.isnan(returns)]
```

Comment:
Oh interesting.  I changed it to use a method that supports both `Series` and `ndarray,` but looks like it's slower on `Series` than `dropna` because `dropna` first converts to using an array.  I'll do the same here.